Monika After Story/game/zz_consumables.rpy

```diff
@@ -1571,6 +1571,10 @@ label mas_consumables_generic_running_out(consumable):
     else:
         m 3eud "I just wanted to let you know that I'm out of [consumable.disp_name][plur]."
 
+    if not renpy.seen_label("mas_consumables_generic_running_out(consumable)"):
```

Comment:
When you use `seen_label`, you pass in only the label name. In this case it should be `renpy.seen_label("mas_consumables_generic_running_out")`. No parameters.

But there's another thing. I think RenPy marks the label as seen so soon as you `jump` to/`call` it. So we will never see this part anyway.

May I suggest to make a new label for that and call it from this and some other labels? It could make sense to tell it from all "out of consumable labels". Like `mas_consumables_generic_queued_running_out_dlg` and `mas_consumables_generic_critical_low`. You'll still need to check `seen_ever` **before** calling it, though.